src/controllers/playback/video/index.js

```diff
@@ -1340,6 +1340,12 @@ export default function (view) {
             case 'PageDown':
                 playbackManager.previousChapter(currentPlayer);
                 break;
+            case 'g':
+                SubtitleSync.prototype.update(v => v - 0.1);
```

Comment:
What if we move these `SubtitleSync.prototype.update(v => v - xxx);` into `SubtitleSync` as functions (say `incrementOffset` and `decrementOffset`)? There we can take a step from the slider - less magic values.

Reply:
done